specification/redis/resource-manager/Microsoft.Cache/2017-02-01/redis.json

```diff
@@ -1029,10 +1261,48 @@
           "description": "Resource tags."
         }
       },
+      "x-ms-azure-resource": true
+    },
+    "ProxyResource": {
+      "description": "The resource model definition for a ARM proxy resource. It will have everything other than required location and tags",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ]
+    },
+    "TrackedResource": {
+      "description": "The resource model definition for a ARM tracked top level resource",
+      "properties": {
+        "tags": {
```

Comment:
You will have to remove `tags` and `location` properties from `Resource` definition in  order to avoid duplicate definitions for these properties (messes up the inheritance chain in our modeler)

Reply:
Makes sense!